public/static/accordion.css

```diff
@@ -24,3 +24,48 @@
   left: -20px;
   top: -1px;
 }
+.accordion a {
+  color: #003d8e;
+}
+.accordion a:visited {
+  color: #003d8e;
+}
```

Comment:
I don't think the styles below this point (`.help`, `.in-touch` etc.) should be in the `accordion` stylesheet since they are for a different component. Wouldn't we use a Styled Component for this as elsewhere? 

Also, if we do need to put them here can we use a naming standard - BEM or equivalent - not sure if there is already one in use on this project. These styles look too general to me and could clash with others.

Reply:
Resolved